sdk/lusid/utilities/config_keys.json

```diff
@@ -7,6 +7,10 @@
       "env": "FBN_LUSID_API_URL",
       "config": "apiUrl"
   },
+  "flag_to_use_decimal" : {
+        "env": "flag_to_use_decimal",
+        "config": "flag_to_use_decimal"
```

Comment:
All the config keys use camelCase e.g. flagToUseDecimal, I think we should stick with that naming convention.

Reply:
As per above, on second thought I think we should just remove this completely from here and pass it into the constructor for the ApiClientFactory.